lisp/mastodon-tl.el

```diff
@@ -224,7 +266,7 @@ Return value from boosted content if available."
       (with-current-buffer (current-buffer)
         (let ((inhibit-read-only t))
           (goto-char (point-max))
-          (mastodon-tl--timeline json))))))
+	  (mastodon-tl--timeline json))))))
```

Comment:
I am not sure why it has shifted this line. In my repo it is inline with the one above.